04.class/class.js

```diff
@@ -0,0 +1,119 @@
+const fs = require('fs')
+const { MultiSelect } = require('enquirer')
+const minimist = require('minimist')(process.argv.slice(2))
+
+const reader = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout
+})
+
+class Memo {
+  constructor () {
+    this.files = fs.readdirSync('./memo_data', 'utf-8')
+  }
+
+  buildChoice () {
+    return this.files.map((path) => {
+      const arrayOfContents = this.buildArrayOfContents(path)
+      const choice = { name: `${arrayOfContents[0]} in ${path}`, value: path }
+      return choice
+    })
+  }
+
+  buildArrayOfContents (filename) {
+    const fileContent = fs.readFileSync(`./memo_data/${filename}`, 'utf-8')
+    const arrayOfContents = fileContent.split(/\r\n|\r|\n/)
+    return arrayOfContents
+  }
+
+  create () {
+    const inputs = []
+    reader.on('line', function (line) {
+      inputs.push('\n' + line)
+    })
+
+    reader.question('Make a note !\n', (answer) => {
+      inputs.push(answer)
+    })
+
+    reader.on('close', function () {
+      const timeStamp = new Date().getTime()
+      fs.writeFileSync(`./memo_data/${timeStamp}.txt`, inputs.join(''))
+      console.log('Your note was saved safely.')
+    }
+    )
+  }
+
+  destroy () {
+    if (this.files.length === 0) {
+      console.log('No files to delete')
+      return
+    }
+
+    const firstLineAndPaths = this.buildChoice()
```

Comment:
```suggestion
    const choices = this.buildChoice()
```

ですかねー

[ユーザーを見つけたら……Aさん「自転車に入れます」Bさん「人間に入れます」Cさん「ユーザーに入れます」 \| FBC](https://bootcamp.fjord.jp/pages/readable-variable-name)

Reply:
@JunichiIto さんレビューありがとうございます！

関数名と変数名が噛み合っていませんね💦

こちらのコミットで修正しましたのでご確認よろしくお願いします。
[6df6741](https://github.com/OdenTakashi/js-practices/pull/4/commits/6df6741d64173f94f616072bfe74f0e8d50055e5)